### ISSUE TITLE
feat(android, ios, web): Right to Left Issues in touch keyboards

### ISSUE THREAD

**Issue by rmlockwood** [keymanapp/keyman#2713]
- [ ] 1. When doing a long press with a right to left script, the additional keys should be to the left of the key in question. Right now additional keys are shown to the right of the key in question. iOS does this correctly in, for example the Arabic keyboard.
- [x] 2. I would like the option of having an opposite facing arrow for the backspace. iOS has this.
- [x] 3. I would like the option of having an opposite facing arrow for the enter symbol. iOS has this.

**Comment by mcdurdin** [comment]
Thank you for your request!

When it comes to implementation, I think we should split this into several separate issues:
* (1) The long-press implementation will impact iOS, Android and web platforms -- one issue each
* (2) and (3) The symbol changes require an update to the OSK font to include opposite facing symbols -- these two changes can be covered in a single issue.

**Comment by jvgaultney** [comment]
I can update the font, but please review this issue to confirm that I understand what you need added:
https://github.com/silnrsi/font-keymanweb-osk/issues/1

**Comment by jahorton** [comment]
After #3851, the main remaining aspect for this would be the default alignment of popup keys.  The base key is no longer shown, so there's no "left" or "right" of the base key to talk about... except for which side of the key the subkeys "spill" into by default.

Though, even then, I think they 'center' by default, so this may actually be resolved in full thanks to those changes.  Once #3851 fully merges, of course.

**Comment by jahorton** [comment]
We've now addressed 2/3rd of this; we'll do a final pass over the 1st point in a future version, though even that has already been moderately addressed as well.

**Comment by darcywong00** [comment]
This can be closed as the 1st point is no longer applicable as we've removed the base key from the popup keys.

The keyboard creator determines the left-to-right order of the popup keys.
